### PR TITLE
Update README with information about End of Cycle feature flags

### PR DESCRIPTION
**Pull request by TobyRet**
### Context
There are three feature flags relating to the end of cycle process. 

### Changes proposed in this pull request
Update README with instructions on how to toggle End of Cycle feature flags locally and on a deployed environment

### Guidance to review
Feedback welcome! Make sense?

### Trello card
https://trello.com/c/38YrjNzp/2079-add-information-about-end-of-cycle-feature-flags-to-readme

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally
- [ ] Product review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,3 +54,51 @@ or
...
+```
+
+To enable this feature on a deployed environment, the following environment variable needs to be updated to `true` in Azure.
+
+```
+SETTINGS__CYCLE_ENDING_SOON
+```
+
+#### 2. Disable 'apply for this course' button
+Turning this flag off removes the 'apply for this course button' on each course listing.
```
Comment (paulrobertlloyd) [inline README.md:76]:
```suggestion
Turning this flag off removes the ‘Apply for this course button’ on each course listing.
```

File: README.md [hunk 2]
```diff
@@ -54,3 +54,51 @@ or
...
+display_apply_button: false
+```
+
+To disable this feature on a deployed environment, the following environment variable needs to be updated to `false` in Azure.
+
+```
+SETTINGS__DISPLAY_APPLY_BUTTON
+```
+
+#### 3. 'Cycle has ended' interim page
```
Comment (paulrobertlloyd) [inline README.md:90]:
```suggestion
### Enable ‘Cycle has ended’ interim page 
```

File: README.md [hunk 3]
```diff
@@ -54,3 +54,51 @@ or
...
+
+To disable this feature on a deployed environment, the following environment variable needs to be updated to `false` in Azure.
+
+```
+SETTINGS__DISPLAY_APPLY_BUTTON
+```
+
+#### 3. 'Cycle has ended' interim page 
+
+Turning this flag on redirects all home, search, results and course page requests to '/cycle-has-ended'.
```
Comment (paulrobertlloyd) [inline README.md:92]:
```suggestion
Turning this flag on redirects all home, search, results and course page requests to `/cycle-has-ended`.
```

File: README.md [hunk 4]
```diff
@@ -54,3 +54,51 @@ or
...
+
+To enable this feature locally, add the following to `config/settings/development.local.yml`
+
+```
```
Comment (paulrobertlloyd) [inline README.md:96]:
```suggestion
```yaml
```

**Review by paulrobertlloyd — CHANGES_REQUESTED** [review]
Just a few copy and markdown syntax tweaks, but otherwise all makes sense to me 😀

**Comment by paulrobertlloyd** [comment]
@TobyRet Any reason for not using smart quotes? (Pet peeve!)

**Comment by TobyRet** [comment]
> @TobyRet Any reason for not using smart quotes? (Pet peeve!)

I've never been asked to follow this convention before. Makes sense though - have made the  changes.
Is there a style guide I'm not aware of?

**Comment by paulrobertlloyd** [comment]
I’m not sure if there’s any style mandated in a GOV.UK style guide (if not, there should be), and too many services get it wrong, but it’s really just a case of using the right characters – using ‘dumb quotes’ is a bit like using `O` in place of `0`. This is a good summary: <https://smartquotesforsmartpeople.com>

**Review by paulrobertlloyd — APPROVED** [review]

**Comment by TobyRet** [comment]
> https://smartquotesforsmartpeople.com

Great link - thanks @paulrobertlloyd

**Pull request merged**